SDK/AppCenterDistribute/Microsoft.AppCenter.DistributePlay/Microsoft.AppCenter.DistributePlay.csproj

```diff
@@ -0,0 +1,19 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<Project Sdk="MSBuild.Sdk.Extras/2.0.54">
+  <Import Project="..\Microsoft.AppCenter.Distribute.Shared\Microsoft.AppCenter.Distribute.Shared.projitems" Label="Shared" />
+  <PropertyGroup>
+    <TargetFramework>monoandroid70</TargetFramework>
```

Comment:
We already have empty implementation for .NET Standard, please reuse that sources and project file